multiplexer/abci/abci.go

```diff
@@ -181,6 +181,24 @@ func (m *Multiplexer) VerifyVoteExtension(_ context.Context, req *abci.RequestVe
 	return app.VerifyVoteExtension(req)
 }
 
+func (m *Multiplexer) QuerySequence(ctx context.Context, req *abci.RequestQuerySequence) (*abci.ResponseQuerySequence, error) {
+	// Return noop (sequence 0) for app version <= 4
+	if m.appVersion <= 4 {
+		return &abci.ResponseQuerySequence{Sequence: 0}, nil
+	}
```

Comment:
here's the defensive noop for not operating this method for every version of the application that doesn't need to run it.

we'd probably be fine anyway, but might as well add it